include/my_cpu.h

```diff
@@ -81,7 +94,13 @@ static inline void MY_RELAX_CPU(void)
   __asm__ __volatile__ ("pause");
 #endif
 #elif defined(_ARCH_PWR8)
-  __ppc_get_timebase();
+#ifdef __FreeBSD__
+  uint64_t __tb;
+  __asm__ volatile ("mfspr %0, 268" : "=r" (__tb));
+#else
+  /* Changed from __ppc_get_timebase for musl compatibility */
+  __builtin_ppc_get_timebase();
+#endif
```

Comment:
Is there a reason why we use inline assembler on FreeBSD and a built-in function elsewhere? On https://godbolt.org there only is a clang trunk for POWER. It would translate `__builtin_ppc_get_timebase()` to the same instruction, with 268 as the second operand.

This came across my head while I was working on #3752.

Reply:
I think I was just copying the FreeBSD patch. With it in LLVM it seems unlikely it was required, or if so it was a very long time ago.

https://github.com/llvm/llvm-project/commit/65e1e4dbe0f1fff23eceeb0fcadd93e8f0a302aa